frontend/src/constants/config.js

```diff
@@ -34,6 +38,32 @@ const settings = {
         default: process.env.NODE_ENV === 'development',
         required: true,
     },
+    PLATFORM_OWNER_NAME: {
+        value: process.env.REACT_APP_PLATFORM_OWNER_NAME || 'Junction',
+        required: true,
+    },
+    PLATFORM_OWNER_WEBSITE: {
+        value:
+            process.env.REACT_APP_PLATFORM_OWNER_WEBSITE ||
+            'https://hackjunction.com',
+        required: true,
+    },
+    SEO_PAGE_TITLE: {
+        value: process.env.REACT_APP_SEO_PAGE_TITLE,
+        required: true,
+    },
+    SEO_PAGE_DESCRIPTION: {
+        value: process.env.REACT_APP_SEO_PAGE_DESCRIPTION,
+        required: true,
+    },
+    SEO_IMAGE_URL: {
+        value: process.env.REACT_APP_SEO_IMAGE_URL || '',
+        required: false,
+    },
+    SEO_TWITTER_HANDLE: {
+        value: process.env.REACT_APP_SEO_TWITTER_HANDLE || '',
+        required: false,
+    },
```

Comment:
Can you provide the previous values for these as defaults? Same way as you did for config.PLATFORM_OWNER_NAME. That way I can just merge it without having to worry about breaking our production deployment :)